src/util/misc/objectEnlive.ts

```diff
@@ -88,13 +94,14 @@ export const enlivenObjects = <
     Promise.all(
       objects.map((obj) =>
         classRegistry
-          .getClass(obj.type)
-          .fromObject(obj, {
-            signal,
-            reviver,
```

Comment:
ok i wonder how reviver ended up here in the refactors

Reply:
just because of the type change

Reply:
yes but it never was part of fromObject, why did we put it there

Reply:
a good question for the blame and spare time to answer